lib/table.js

```diff
@@ -72,31 +72,26 @@ Table.createFeatureTable = function (id, geojson, layerId, callback) {
   })
 }
 
-  /**
-   * Creates a new table
-   * checks to see if the table exists, create it if not
-   *
-   * @param {string} name - the name of the index
-   * @param {string} schema - the schema to use for the table
-   * @param {Array} indexes - an array of indexes to place on the table
-   * @param {function} callback - the callback when the query returns
-   * @private
-   */
+/**
+ * Creates a new table
+ * checks to see if the table exists, create it if not
+ *
+ * @param {string} name - the name of the index
+ * @param {string} schema - the schema to use for the table
+ * @param {Array} indexes - an array of indexes to place on the table
+ * @param {function} callback - the callback when the query returns
+ * @private
+ */
 Table.create = function (name, schema, indexes, callback) {
   var self = this
   // set callback to noop if it hasn't been passed in
   callback = callback || function () {}
-  var sql = "select exists(select * from information_schema.tables where table_name='" + name + "')"
-  self.query(sql, function (err, result) {
-    if (err) return callback(new Error('Failed to create table ' + name))
-    if (result && result.rows[0].exists) return callback()
-    var create = 'CREATE TABLE "' + name + '" ' + schema
-    self.query(create, function (err, result) {
-      if (err) return callback(new Error('Failed to create table ' + name + ' error:' + err))
-      if (!indexes || !indexes.length) return callback()
-      Indexes._add(name, indexes, function (err) {
-        callback(err)
-      })
+  var create = 'CREATE TABLE "' + name + '" ' + schema
+  self.query(create, function (err, result) {
+    if (err) return callback(new Error('Failed to create table ' + name + ' error:' + err))
```

Comment:
do we get any special code on failed to create because table exists? can we do a create if not exists?

Reply:
Postgres error code `42P07` - but I am able to add the `IF NOT EXISTS` (will add a test).